src/renderer/components/ft-video-player/ft-video-player.css

```diff
@@ -5,3 +5,9 @@
 .ftVideoPlayer {
   max-height: 50vh;
 }
+
+.relative.vjs-overlay {
```

Comment:
Styling for the video player itself is a little odd, in that the styling isn't done within this component.  The styling for the video player is within `videoJS.css` which is located in the root of the `renderer` folder.

In this file (preferably towards the bottom), we'll want to add the following block.

```css
.vjs-overlay {
  font-size: xx-large;
  max-width: 100% !important;
}
```

The `!important` tag has to be included because the `vjs-overlay` CSS is forcing the max-width to be 33%, so this makes sure that this is overridden.

Once this has been done, you can completely remove the `.relative.vjs-overlay` block within `ft-video-player.css`